README.md

```diff
@@ -41,22 +41,20 @@ More instructions on installing LXD for a wide variety of Linux distributions an
 
 To install LXD from source, see [Installing LXD](doc/installing.md) in the documentation.
 
-<!-- Include start security -->
-
 ## Security
-LXD, similar to other container and VM managers provides a UNIX socket for local communication.
 
-**WARNING**: Anyone with access to that socket can fully control LXD, which includes
```

Comment:
I think I'd like to keep the prominent warning in the README.md, this is more visible than the bulletpoints and I think needs to be seen without having to go to the separate security doc.

Reply:
Added back.